_posts/en/newsletters/2019-05-14-newsletter.md

```diff
@@ -0,0 +1,485 @@
+---
+title: 'Bitcoin Optech Newsletter #46'
+permalink: /en/newsletters/2019/05/14/
+name: 2019-05-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## Action items
+
+FIXME
+
+## Dashboard items
+
+FIXME: highish feerates
+
+## News
+
+FIXME
+
+## Overview of the Taproot & Tapscript proposed BIPs
+
+Last week, Pieter Wuille [emailed][] the Bitcoin-Dev mailing list with
+links to two proposed BIPs.  The first, [bip-taproot][], permits
+spending using either a Schnorr-style signature or merklized script. The
+second, [bip-tapscript][], defines a slight variation on Bitcoin's
+existing Script language to be used in bip-taproot merkle spends.
+
+For readers already familiar with Bitcoin scripting and ideas like
+[MAST][], it should be possible to understand the BIPs directly.  For
+readers with less background, we've prepared the following summary of
+some key features from the proposals by looking at them from the point
+of view of an existing wallet that wants to upgrade to use Taproot and
+Tapscript.  This only skims the surface of what these proposals make
+available, as one reason developers have been waiting for Schnorr
+signatures and MAST-based encumbrances is that they provide the building
+blocks for new features that were previously difficult or impossible to
+build on Bitcoin.  We'll leave the description of those advances for
+another time and focus here on how the two proposed BIPs can make
+existing uses of Bitcoin work even better than they do today.
+
+### What's not in the proposals
```

Comment:
I do think it breaks the flow of the section a bit to put it up front---but it would also break the flow a bit no matter where we put it.  The advantage of the current placement is that is clears up some misconceptions before we get into details and it allows the section to end on a high note.